### ISSUE TITLE
After some period of time, plugin stops working

### ISSUE THREAD

**Issue by amiantos** [bartdorsey/homebridge-blinkcameras#23]
This has happened to me a couple times now, the plugin appears to stop working, not much in the logs to go on. Homebridge is still running fine, but the plugin can't update my devices anymore. I have to restart Homebridge to jog its memory.

```
[5/21/2020, 11:51:38 PM] Homebridge is running on port 51862.
[5/21/2020, 11:51:50 PM] [Blink System] Setup Blink System
[5/21/2020, 11:51:57 PM] [Blink System] [Living Room Camera] is disarmed
[5/21/2020, 11:51:57 PM] [Blink System] [Durango System] is disarmed
[5/22/2020, 12:18:48 AM] [Blink System] [Durango System] is disarmed
[5/22/2020, 12:18:49 AM] [Blink System] [Living Room Camera] is disarmed
[5/22/2020, 1:00:45 AM] [Blink System] [Durango System] is disarmed
[5/22/2020, 1:00:45 AM] [Blink System] [Living Room Camera] is disarmed
[5/22/2020, 1:04:58 AM] [Blink System] [Durango System] is disarmed
[5/22/2020, 1:04:58 AM] [Blink System] [Living Room Camera] is disarmed
[5/22/2020, 11:12:48 PM] [Blink System] [Living Room Camera] is disarmed
[5/22/2020, 11:12:48 PM] [Blink System] [Durango System] is disarmed
[5/22/2020, 11:49:00 PM] [Blink System] [Durango System] is disarmed
[5/22/2020, 11:49:01 PM] [Blink System] [Living Room Camera] is disarmed
[5/23/2020, 12:10:52 AM] [Blink System] Authenticating with Blink API as b******t@me.com
[5/24/2020, 8:01:49 AM] [Blink System] Authenticating with Blink API as b******t@me.com
```

Once it gets into that "Authenticating..." state there is no recovery, gotta restart it.

**Comment by SchneidoDUS** [comment]
I have the same issue. Could you solve it already?

**Comment by amiantos** [comment]
Nope. Really seems like Blink is trying to kill off all third party plugins.

**Comment by amiantos** [comment]
I have a feeling this comment is a clue to this issue: https://github.com/bartdorsey/homebridge-blinkcameras/issues/17#issuecomment-637806487

**Comment by lovie** [comment]
I have the same issue, after the 24 hour re-auth, the plugin freezes my home app too. I have added a fix by adding a method call “this.discover()” after the 24 hour reauth completes at line number 95 in index.js (getblink method). That fixed it for me.

**Comment by lovie** [comment]
Pull request submitted.

**Comment by bartdorsey** [comment]
Thanks for the PR, I've fixed the problem in the latest release v1.1.0.  I'm re-authenticating every 24 hours now to get a new token.
